.golangci.yaml

```diff
@@ -0,0 +1,135 @@
+run:
+  timeout: 4m
+
+linters:
+  disable-all: false
+  enable:
+    - gocyclo
+    - misspell
+    - nilerr
+    - unparam
+    - gosec
+    - unused
+    - govet
+    - gosimple
+    - errorlint
+    - errcheck
+    - dogsled
+    - cyclop
```

Comment:
There's a certain intersection between what `gocyclo` and `cyclop` offer. The latter seems to no longer be maintained, so maybe we need only `gocyclo`.